spring-data-sample/README.md

```diff
@@ -0,0 +1,262 @@
+# Spring Data integration Sample
+
+This is a sample Spring Boot application for Spring Data integration with ScalarDB.
+
+## Prerequisites
+- Java (OpenJDK 8 or higher)
+- Gradle
+- Docker, Docker Compose
+
+## Sample application
+
+### Overview
+
+This article describes how to create a Spring Boot applicaiton for the same use case as [ScalarDB Sample](https://github.com/scalar-labs/scalardb-samples/tree/main/scalardb-sample) using Spring Data integration with ScalarDB.
+Please note that application-specific error handling, authentication processing, etc., are omitted in the sample application since it focuses on explaining how to use Spring Data integration.
+Please see [this document](https://scalar-labs.github.io/scalardb-sql/spring-data-guide.html) for the details.
+
+### Schema
+
+[The schema](schema.json) is as follows:
+
+```json
+{
+  "sample.customers": {
+    "transaction": true,
+    "partition-key": [
+      "customer_id"
+    ],
+    "columns": {
+      "customer_id": "INT",
+      "name": "TEXT",
+      "credit_limit": "INT",
+      "credit_total": "INT"
+    }
+  },
+  "sample.orders": {
+    "transaction": true,
+    "partition-key": [
+      "customer_id"
+    ],
+    "clustering-key": [
+      "timestamp"
+    ],
+    "secondary-index": [
+      "order_id"
+    ],
+    "columns": {
+      "order_id": "TEXT",
+      "customer_id": "INT",
+      "timestamp": "BIGINT"
+    }
+  },
+  "sample.statements": {
+    "transaction": true,
+    "partition-key": [
+      "order_id"
+    ],
+    "clustering-key": [
+      "item_id"
+    ],
+    "columns": {
+      "order_id": "TEXT",
+      "item_id": "INT",
+      "count": "INT"
+    }
+  },
+  "sample.items": {
+    "transaction": true,
+    "partition-key": [
+      "item_id"
+    ],
+    "columns": {
+      "item_id": "INT",
+      "name": "TEXT",
+      "price": "INT"
+    }
+  }
+}
+```
+
+All the tables are created in the `sample` namespace.
+
+The `sample.customers` table manages customers' information.
+The `credit_limit` is the maximum amount of money a lender will allow each customer to spend using a credit card, and the `credit_total` is the amount of money that each customer has already spent by using the credit card.
+
+The `sample.orders` table manages orders' information, and the `sample.statements` table manages the statements' information of the orders. Finally, the `sample.items` table manages items' information to be ordered.
+
+
+The ER diagram for the schema is as follows:
```

Comment:
Does the acronym "ER" stand for "entity-relationship"? If so, how about we spell out the acronym instead?